src/overlays/actors/ovl_Bg_Ice_Shelter/z_bg_ice_shelter.c

```diff
@@ -1,18 +1,27 @@
+/*
+ * File: z_bg_ice_shelter.c
+ * Overlay: Bg_Ice_Shelter
+ * Description: Red Ice
+ */
+
 #include "z_bg_ice_shelter.h"
 #include "assets/objects/object_ice_objects/object_ice_objects.h"
 
 #define FLAGS 0
 
+#define BGICESHELTER_GET_TYPE(thisx) (((thisx)->params >> 8) & 7)
+#define BGICESHELTER_USING_SWITCH_FLAG(thisx) (!(((thisx)->params >> 6) & 1))
```

Comment:
Im not totally sure about including the ! in the macro. I agree that it does make it easier to name, since its nicer to say that it *does* use a switch flag. but yeah I feel like these params getters should probably just be about getting the params and not introducing any more extra logic. would be curious on others opinion

Reply:
I don't have very strong feelings when it's a boolean like this, but `NO_SWITCH_FLAG` seems like it would work just as well.

Reply:
Thanks for the feedback. I have renamed the macro to `BGICESHELTER_NO_SWITCH_FLAG`.